tasks/fh-build.js

```diff
@@ -43,7 +41,7 @@ module.exports = function(grunt) {
    * If grunt config contains an array property called 'fhignore',
    * its elements will be excluded from the tarball.
    */
-  var mergePatterns = function(bundle_deps) {
+  var mergePatterns = function (bundle_deps) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)